bin/tictactoe

```diff
@@ -1,3 +1,14 @@
 #!/usr/bin/env ruby
 
 require_relative '../config/environment'
+
+puts "Welcome to tic tac toe! How many players wish to play?"
+num_players = gets.strip.to_i
+if num_players == 0
+  game = Game.new(Players::Computer.new("X"), Players::Computer.new("O"), Board.new)
+elsif num_players == 1
+  game = Game.new(Players::Human.new("X"), Players::Computer.new("O"), Board.new)
+elsif num_players == 2
+  game = Game.new
+end
+game.play
```

Comment:
Prefer to encapsulate CLI logic into a class. For example, the only code in your bin executable should be:

```ruby
#!/usr/bin/env ruby
       
require_relative '../config/environment'

TicTacToeCLI.new.call
```

And then you'd have a file: `lib/tic_tac_toe_cli.rb` with:

```ruby
class TicTacToeCLI
  def initialize
  end

  def call
   # Your game logic
  end
end
```

The code in your bin would generally just go in `call`.